pkg/utils/reader.go

```diff
@@ -72,6 +89,64 @@ func InitFileReader() *FileReader {
 	return &FileReader{}
 }
 
+type msg struct {
+	Msg         string
+	DomainName  string
+	Hostname    string
+	HTTPStatus  int
+	Batch       string
+	Region      string
+	Latency     int
+	Az          string
+	Url         string
+	UserAgent   string
+	IPv4Address string
+	Port        int
+}
```

Comment:
I don't think you need this struct, it seems like it's just causing almost duplicated code in the `createK8sBody()` function, and isn't used anywhere else.